community/standards/index.html

```diff
@@ -19,7 +19,7 @@
       <h3>Be respectful and inclusive</h3>
       <p>
         Please do not use overtly sexual language or imagery, and do not attack anyone based on any aspect of personal identity,
-        including gender, sexuality, religion, ethnicity, race, age or ability. Keep in mind that what you write in public forums is
+        including gender, sexuality, politics, religion, ethnicity, race, age, ability, etc. Keep in mind that what you write in public forums is
```

Comment:
"etc" doesn't seem quite direct enough, IMO; I'd say "or anything else." Does that seem reasonable?